src/OpenTelemetry/OpenTelemetry.csproj

```diff
@@ -26,4 +26,10 @@
     <Compile Include="$(RepoRoot)\src\Shared\ThreadSafeRandom.cs" Link="Includes\ThreadSafeRandom.cs" />
   </ItemGroup>
 
+  <PropertyGroup>
+  <PackageReadmeFile>README.md</PackageReadmeFile>
+  <PackageReleaseNotes>RELEASENOTES.md</PackageReleaseNotes>
+</PropertyGroup>
```

Comment:
nit:

```suggestion
    <PackageReadmeFile>README.md</PackageReadmeFile>
    <PackageReleaseNotes>RELEASENOTES.md</PackageReleaseNotes>
  </PropertyGroup>
```

Reply:
Should we move this to the props file so it applies to all packages?

In here: https://github.com/open-telemetry/opentelemetry-dotnet/blob/19e1663c1e9485948d522d4f6d77ab652d6033bc/build/Common.prod.props#L13

Thinking something like...

```diff
    <PackageThirdPartyNoticesFile>$(RepoRoot)\THIRD-PARTY-NOTICES.TXT</PackageThirdPartyNoticesFile>
+   <PackageReadmeFile>README.md</PackageReadmeFile>
+   <PackageReleaseNotes>$(RepoRoot)\RELEASENOTES.md</PackageReleaseNotes>
  </PropertyGroup>
```